pkg/flag.go

```diff
@@ -11,18 +11,23 @@ type KeyValuePair struct {
 	Value string
 }
 
-// KeyValuePairs is a slice of KeyValuePair.
+// KeyValuePairs is the slice type for KeyValuePair
```

Comment:
I think my comment is a bit clearer, isn't it?

Reply:
Sure, I put it back